superset/views/core.py

```diff
@@ -3054,6 +3083,259 @@ def schemas_access_for_csv_upload(self):
                 stacktrace=utils.get_stacktrace(),
             )
 
+    @api
+    @has_access_api
+    @expose("/csvtodatabase/add", methods=["POST"])
```

Comment:
Would be nice to keep a separate view for `csvtodatabase` routes. This file is already thousands of lines long and we should be breaking it up instead of adding to it.

Reply:
I concur - ideally, we should add this to the `/api/v1` routes since it is an API.